web/themes/custom/nysenate_theme/src/patterns/global/colors/_colors.scss

```diff
@@ -90,7 +90,7 @@ $orange01: #cb8d37; // rgb(203, 141, 55)
 $blue01: #2b6990; // rgb(43, 106, 144)
 $blue02: #004c71; // rgb(0, 77, 113) - or there about
 $blue03: #04a9c5;
-$yellow01: #f1af58; //rgb(241, 175, 88);
+$yellow01: #AC6202; //rgb(241, 175, 88);
```

Comment:
don't forget to update the comments  :)

Reply:
yup, i just wanted to get the branch established, quick change to get going